src/components/Button/components/ButtonBase.tsx

```diff
@@ -238,27 +217,26 @@ const StyledButton = styled.button<IStyledBaseButton>`
   span {
     margin-right: ${props => getIconMargin(props)};
     svg path {
-      fill: ${props => getIconColor("default", props)};
+      fill: ${props => getFontColor("default", props)};
     }
   }
   :hover,
   :active,
   :focus {
+    color: ${props => getFontColor("hover", props)};
     background-color: ${props => getBackgroundColor("hover", props)};
     svg path {
-      fill: ${props => getIconColor("hover", props)};
+      fill: ${props => getFontColor("hover", props)};
     }
   }
   :active,
   :focus {
-    border: ${props => getBorder("active", props)};
-    svg path {
-      fill: ${props => getIconColor("focus", props)};
-    }
+    border: ${props => getBorder("active", props)} !important;
+    box-shadow: 0px 0px 2px hsl(216, 100%, 50%);
```

Comment:
The shadow are global for focus. Can we define it as a constant...or something like it's done for border?

Reply:
Fixed